src/routes/auth.route.ts

```diff
@@ -18,24 +18,20 @@ const authRouter = express.Router();
 // );
 
 authRouter.route("/register").post(async (req: Request, res: Response) => {
-  // try {
-  // process input data
-  // call repository method
   const data = req.body;
-
   const result = await register(data);
   res.send(result);
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  // } catch (error: any) {
-  //   let message;
-  //   if (error instanceof Error) {
-  //     message = error.message;
-  //   } else {
-  //     message = String(error);
-  //   }
-  //   console.error(error);
-  //   res.status(400).send(message);
-  // }
+});
+
+authRouter.route("/login").post(async (req: Request, res: Response) => {
+  try {
+    const data = req.body;
+    const result = await login(data);
+    res.send(result);
+  } catch (error: unknown) {
+    console.error(error);
+    return res.status(400).send(error.toString());
+  }
```

Comment:
```suggestion
  }
```
You can remove try..catch now it catches in middlewares